### DISCUSSION TITLE
Can't see the SR while adjusting mods

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#18401] [Ideas] [answered]
I like to use lazer for special difficulty adjusting. For example, I have a 6.3* map, and I am using DT on it and changing the value to specific what turns it into 7* map (like 1.18x speed-up), but with the new interface, I must quit adjusting the menu to check SR. The new interface looks good and fresh, but this problem makes my diff adjustment very frustrating.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/18189
